tests/models/test_main.py

```diff
@@ -20,26 +20,26 @@
 from tests.helpers import FixtureMixin
 
 
-def test_should_not_return_password_on_user():
-    app = create_app('test')
-    now = datetime.utcnow()
-    user = User(
-        email_address='email@digital.gov.uk',
-        name='name',
-        role='buyer',
-        password='password',
-        active=True,
-        failed_login_count=0,
-        created_at=now,
-        updated_at=now,
-        password_changed_at=now
-    )
+class TestUser(BaseApplicationTest, FixtureMixin):
+    def test_should_not_return_password_on_user(self):
+        with self.app.app_context():
+            now = datetime.utcnow()
+            user = User(
+                email_address='email@digital.gov.uk',
+                name='name',
+                role='buyer',
+                password='password',
+                active=True,
+                failed_login_count=0,
+                created_at=now,
+                updated_at=now,
+                password_changed_at=now
+            )
 
-    with app.app_context():
-        assert_equal(user.serialize()['emailAddress'], "email@digital.gov.uk")
-        assert_equal(user.serialize()['name'], "name")
-        assert_equal(user.serialize()['role'], "buyer")
-        assert_equal('password' in user.serialize(), False)
+            assert_equal(user.serialize()['emailAddress'], "email@digital.gov.uk")
```

Comment:
I believe we are trying to remove nose functions from all our tests suites so we just use `assert user.serialize()['emailAddress'] ==  "email@digital.gov.uk"` instead. I think we had removed these from nearly everywhere but these might be some final ones if you'd like to do that whilst your at it please?

Reply:
Yep, happy to change. This test was clearly a bit of an outlier!